java_common.gradle

```diff
@@ -70,6 +70,26 @@ configurations {
   }
 }
 
+// Custom resolution strategy to get around tricky dependency issues.
+configurations.all {
```

Comment:
Add a TODO to remove this once we migrate to Gradle 7.6+ ?  (Or whatever the relevant condition is?)